cf-deploy-teardown/tasks/cf-deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+#export k8s-host details from pool
+set -a; source pool.k8s-hosts/metadata; set +a
+
+#check k8s host readiness to deploy CF
+ssh-keygen -N "" -f /root/.ssh/id_rsa
+sshpass -p "$K8S_PASSWORD" ssh-copy-id -o StrictHostKeyChecking=no ${K8S_USER}@${K8S_HOST_IP}
```

Comment:
@prabalsharma We should avoid displaying the password here as it shows up in the Concourse logs. Apparently `sshpass` can take the password from the environment with the `-e` option, so the K8S_PASSWORD env var could be changed in the metadata file to `SSHPASS`.